src/Symfony/Component/Form/Extension/Core/Type/DateTimeType.php

```diff
@@ -237,6 +237,11 @@ public function configureOptions(OptionsResolver $resolver)
             'compound' => $compound,
             'date_label' => null,
             'time_label' => null,
+            'invalid_message' => function(Options $options, $previousValue) {
```

Comment:
I believe the proper cs here is:

```diff
- function(Options $options, $previousValue) {
+ function (Options $options, $previousValue) {
```

There are a lot more occurrences. I thought fabbot would have smelt these.